src/util.ts

```diff
@@ -69,3 +69,9 @@ export function getToolcachePath(toolName: string, version: string, architecture
 
   return null;
 }
+
+export function isJobStatusSuccess() {
+  const jobStatus = core.getInput('job-status');
```

Comment:
Could you also add `job-status` as a constant to `src/constants.ts` (maybe `INPUT_JOB_STATUS`). Minor but lets follow the pattern we have for all inputs.